specs/src/specs/params.md

```diff
@@ -0,0 +1,50 @@
+# Celestia Governance Params
+
+These are the parameters for mainnet. Note that not all of these parameters are
+changable via governance. This list also includes parameter that require a
+hardfork to change due to being manually hardcoded in the application or they
+are blocked by the `x/paramfilter` module.
+
+## Parameters
+
+| Module.Parameter | Default | Summary | Changeable via Governance |
```

Comment:
General remark: The `Default` columns should be completely consistent! Currently it is sometimes `0.05 (5%)` (like the Minimum commission rate) but sometimes only `0.02` (like the community pool). 
I personally think the default value should first be in the column in the unit the param is passed and then explained in a more human friendly way. The comment should explicitly mention the unit this is passed into the param. See my comment for MaxAgeDuration for a concrete example: https://github.com/celestiaorg/celestia-app/pull/2416/files#r1315177546

Reply:
updated mainly here https://github.com/celestiaorg/celestia-app/pull/2416/commits/debdc9b69e250190f3846260b32a91c037916484